src/config.ts

```diff
@@ -22,9 +22,7 @@ export async function readUserConfig(configPath: string): Promise<Config> {
   try {
     return require(configPath);
   } catch (err: any) {
-    if (err.code !== "MODULE_NOT_FOUND") {
-      console.error(err);
-    }
+    console.error(err);
```

Comment:
This would even show the error when a user doesnt have a etsc.config file ? 
users can use the cli without a etsc config file too

Reply:
True, I just fixed that, and now you'll get an error only if the file exists. otherwise it will load the default config